src/lambda/commands/createNewSamApp.ts

```diff
@@ -81,15 +84,14 @@ export async function createNewSamApp(
 }
 
 async function getMainUri(config: Pick<SamCliInitArgs, 'location' | 'name'>): Promise<vscode.Uri | undefined> {
-    try {
-        return await getMainSourceFileUri({
-            root: vscode.Uri.file(path.join(config.location.fsPath, config.name))
-        })
-    } catch (err) {
+    const samTemplatePath = path.resolve(config.location.fsPath, config.name, 'template.yaml')
+    if (await fileExists(samTemplatePath)) {
+        return vscode.Uri.file(samTemplatePath)
+    } else {
         vscode.window.showWarningMessage(localize(
             'AWS.samcli.initWizard.source.error.notFound',
             'Project created successfully, but main source code file not found: {0}',
```

Comment:
Change to something like "Project created successfully, but project template file not found: {0}"?

Reply:
Actually, I think if there is no template file the project has not been created successfully.